documentation/overview.md

```diff
@@ -55,153 +56,188 @@ The following scenarios are not supported:
 
 Note that the lzo codec is not available by default in Azure Databricks but must be installed. See
 [https://docs.microsoft.com/en-us/azure/databricks/data/data-sources/read-lzo](https://docs.microsoft.com/en-us/azure/databricks/data/data-sources/read-lzo)
 
-By default, data is written as CSV into a default Data subfolder within the target entity folder.
-
-Options to use and control the partition pattern, and to provide CSV options are not yet supported.
-
 #### Save Mode
 
-The save mode specifies how existing entity data in the CDM folder is handled. Options are to
-overwrite, append to, or error if data already exists. The default save mode is ErrorIfExists
+The save mode specifies how existing entity data in the CDM folder is handled when writing a dataframe. Options are to overwrite, append to, or error if data already exists. The default save mode is ErrorIfExists
 
 |**Mode**  |**Description**|
 |---------|---------|
-|SaveMode.Overwrite |Will overwrite existing partitions with data being written.<br/>Note: overwrite does not support changing the schema; if the schema of the data being written is incompatible with the existing entity definition an error will be thrown. |
-|SaveMode.Append |Will append data being written as new partitions alongside the existing partitions.<br/>Note: append does not support changing the schema; if the schema of the data being written is incompatible with the existing entity definition an error will be thrown.|
+|SaveMode.Overwrite |Will overwrite the existing entity definition if it is changed and replace existing data partitions with the data partitions being written.|
+|SaveMode.Append |Will append data being written in new partitions alongside the existing partitions.<br/>Note: append does not support changing the schema; if the schema of the data being written is incompatible with the existing entity definition an error will be thrown.|
 |SaveMode.ErrorIfExists|Will return an error if partitions already exist.|
 
-See *Folder organization* below for details of how data files are organized on write.
+See *Folder and file organization* below for details of how data files are named and organized on write.
 
 ### Examples
 
 The following examples all use appId, appKey and tenantId variables initialized earlier in the code based on an Azure app registration that has been given Storage Blob Data Contributor permissions on the storage for write and Storage Blob Data Reader permissions for read.
 
+#### Read
+This code reads the Person entity from the CDM folder with manifest in mystorage.dfs.core.windows.net/cdmdata/contacts/root.manifest.cdm.json.
+
+```scala
+val df = spark.read.format("com.microsoft.cdm")
+ .option("storage", "mystorage.dfs.core.windows.net")
+ .option("container", "cdmdata")
```

Comment:
Same as below comments

Remove container and manifest option and replace with manifestPath - cdmdata/contacts/root.manifest.cdm.json